dts/arm/nxp/nxp_rt6xx_common.dtsi

```diff
@@ -40,6 +40,14 @@
 	};
 };
 
+&systick {
+	/*
+	 * RT600 relies by default on the OS Timer for system clock
```

Comment:
If this is the case, why is os_timer status `disabled` by default?

Reply:
Thanks for catching this, I have fixed it.

Reply:
Since you disable systick at the SoC level, you should also enable os_timer at the SoC level. Currently it's done at the board level

Reply:
It gives each board the flexibility to pick, do we not want this?

Reply:
Each board will still have the flexibility to pick using a dts overlay.